src/main/drivers/serial_uart_impl.h

```diff
@@ -227,6 +233,7 @@ typedef struct uartDevice_s {
 #if !defined(STM32F4) // Don't support pin swap.
     bool pinSwap;
```

Comment:
Noticed bool pinSwap is not used?

Reply:
Out of scope of this change so I'll raise another PR to clean up. I don't believe we ever use DMA either so it's probably safe to prune that out too.